primer-service/test/outputs/OpenAPI/openapi.json

```diff
@@ -318,7 +318,10 @@
                     "DeleteCon",
                     "AddConField",
                     "DeleteConField",
-                    "DeleteTypeParam"
+                    "DeleteTypeParam",
+                    "MakeKType",
+                    "MakeKFun",
+                    "DeleteKind"
```

Comment:
This commit should maybe be a breaking change? We may emit new stuff from the api

Reply:
Yes, you're right.